app/WsConnection.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE TypeFamilies      #-}
+
+{- |Implementation of a connection using Websockets.
+-}
+module WsConnection (WsConnection(..)) where
+
+import           App.ConnectionMgnt
+import           ClassyPrelude
+import qualified Data.Aeson         as Aeson
+import qualified Network.WebSockets as WS
+
+
+newtype WsConnection = WsConnection WS.Connection
+
+instance IsConnection WsConnection where
+    type Pending WsConnection = WS.PendingConnection
+
+    sendMsg (WsConnection conn) =
```

Comment:
The type of sendMsg` is in fact `c -> MessageForClient -> IO ()` for better mocking & to be more disconnected from the Websockets in this place